src/Main.elm

```diff
@@ -28,25 +32,109 @@ init =
     { display = "0"
     , displayStatus = ShowingResults
     , decimalStatus = NoDecimal
+    , valueA = 0
+    , actionA = Blank
+    , valueB = 0
+    , actionB = Blank
     }
 
 
+type DisplayStatus
+    = ShowingResults
+    | NotResults
+
+
+type DecimalStatus
+    = YesDecimal
+    | NoDecimal
+
+
+orderOfOperations : Model -> Model
+orderOfOperations model =
+    case ( model.actionA, model.actionB ) of
+        ( Add, Equals ) ->
+            simpleMath model
+
+        ( Add, Add ) ->
+            simpleMath model
+
+        ( _, _ ) ->
+            model
+
+
+simpleMath : Model -> Model
+simpleMath model =
+    case model.actionA of
```

Comment:
I would probably get rid of the `case` statement entirely here and just name this function something like `executeAdd`